ql/src/test/queries/clientpositive/jdbc_sort_limit.q

```diff
@@ -0,0 +1,132 @@
+--!qt:database:postgres:q_test_tpcds_tables.sql
+
+CREATE EXTERNAL TABLE IF NOT EXISTS `date_dim`(
+  `d_date_sk` int, 
+  `d_date_id` string, 
```

Comment:
I don't understand the purpose of including this file `q_test_tpcds_tables.sql` here.
The same tables that are created in that file are also created in this q-test.

Reply:
`q_test_tpcds_tables.sql` creates them in a postgres database in a docker container.
In this file, we create external tables for the jdbc tables, and eventually use these external tables for running qtests.